book.adoc

```diff
@@ -0,0 +1,276 @@
+= ownCloud Administration Manual
+The ownCloud Developers <docs@owncloud.com>
```

Comment:
admin manual - developers ?

Reply:
Just copying what was there (at least in one of the manuals). Perhaps I was a bit lazy.